src/main/java/no/nav/veilarbvedtaksstotte/service/KafkaConsumerService.java

```diff
@@ -58,6 +65,20 @@ public void behandleEndringPaOppfolgingsbruker(ConsumerRecord<String, KafkaOppfo
         }
     }
 
+    public void behandleEndringPaOppfolgingsbruker(ConsumerRecord<String, KafkaOppfolgingsbrukerEndringV2> kafkaOppfolgingsbrukerEndring) {
+        String fnr = kafkaOppfolgingsbrukerEndring.value().getFodselsnummer();
+        AktorId aktorId = aktorOppslagClient.hentAktorId(Fnr.ofValidFnr(fnr));
```

Comment:
Trenger strengt tatt ikke validere fnr i denne konteksten. Finnes også en `Fnr.of(...)`. Skal også fungere å type fodselsnummer i `KafkaOppfolgingsbrukerEndringV2` til `Fnr` i stede for `String`